src/NeoSharp.Core/Network/IServerContext.cs

```diff
@@ -9,5 +10,9 @@ public interface IServerContext
         /// Version
         /// </summary>
         VersionPayload Version { get; }
+
+        IEnumerable<EndPoint> ConfigurableEndpoints { get; }
```

Comment:
This information is useful only for Server. Other consumers have no point to see this.

Reply:
That is the reason for the ServerContext exist. To keep the DTOs away from the Server

Reply:
My question was if I inject IServerContext to any message handler how can I benefit from ConfigurableEndpoints property? Only IServer knows how to take this and transform to peers through many steps.